### DISCUSSION TITLE
Customizing the RefundProcess

### DISCUSSION THREAD

**Discussion by monrostar** [vendurehq/vendure#2433] [General]
When developing multivendors we faced the problem that we were limited in modifying refunds from multivendors’ channels, if we wanted to limit any refunds that involved local banks and required verification from the main channel. 

I think it would be a good idea to create the same refund process as the Order or Payment to customize the refund flow.

@michaelbromley @martijnvdbrug

**Comment by michaelbromley** [comment]
Thanks for the suggestion. I've converted this into an issue https://github.com/vendure-ecommerce/vendure/issues/2465 so we can track it and discuss it there.
